src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/retry/throttle/impl/MainConsumerThrottleStrategy.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.companieshouse.chipsrestinterfacesconsumer.retry.throttle.impl;
+
+import uk.gov.companieshouse.chipsrestinterfacesconsumer.retry.throttle.ConsumerThrottleStrategy;
+
+public class MainConsumerThrottleStrategy implements ConsumerThrottleStrategy {
+
+    @Override
+    public void throttle() {
+        return;
```

Comment:
sonar isnt a fan of this due to an un needed jump in the code.
Maybe all the delay strategy to be null, and just not have one for the main consumer.
With a if (delayStrategy != null) { delayStrategy.throttle();} for calling delay?